### PR TITLE
Patch fire related genes

### PR DESCRIPTION
**Pull request by Person243546**
## Additions

Adds patch to include PrometheumFlame and Flame_Secondary to the effects of fire related genes (Biotech, Big and Small Genes and Alpha Genes)

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [x] Playtested a colony (1 year in-game)

### CODE REVIEW

File: Biotech/Patches/GeneDefs/GeneDefs_Misc.xml
Comment (albertbokor-unideb) [inline Biotech/Patches/GeneDefs/GeneDefs_Misc.xml:1]: that's.... a looot....

File: Biotech/Patches/GeneDefs/GeneDefs_Misc.xml [hunk 1]
```diff
@@ -44,5 +44,21 @@
...
+		<value>
+			<PrometheumFlame>0.25</PrometheumFlame>
+			<Flame_Secondary>0.25</Flame_Secondary>
+		</value>
+	</Operation>
+	
+	<Operation Class="PatchOperationAdd">
+		<xpath>Defs/GeneDef[defName="FireWeakness"]/damageFactors</xpath>
+		<value>
+			<PrometheumFlame>4</PrometheumFlame>
```
Comment (SerinaCE) [inline Biotech/Patches/GeneDefs/GeneDefs_Misc.xml:59]: that's... a loooot...

Comment (Person243546) [reply Biotech/Patches/GeneDefs/GeneDefs_Misc.xml:59]: It's the same multiplier for vanilla flame. To what valueshould I reduce it?

**Review by N7Huntsman — APPROVED** [review]
Matches vanilla and makes sense.

**Pull request merged**